rpc/README.md

```diff
@@ -2552,7 +2552,7 @@ Subscribe to a topic, if successful it returns the subscription id. For each eve
 
 #### Parameters
 
-* topic - Subscription topic (enum: new_tip_header | new_tip_block)
+* topic - Subscription topic (enum: new_tip_header | new_tip_block | new_transaction)
```

Comment:
It's better to provide more information: what kinds of transactions will be sent if I subscribe "new_transaction"? Because it determines whether this interface is suitable for users to their scenario.
  - transactions submitted into the pending pool via RPC
  - transactions relayed from peers
  - transactions committed within blocks
  - transactions stay in the orphan_pool
  - maybe others

If it only subscribes to the first two, then I prefer renaming "new_pool_transaction".

Reply:
updated the rpc doc, please review again, thanks